samples/Azure.Storage.Tables/readme.md

```diff
@@ -6,9 +6,9 @@
 ``` yaml
 title: Azure.Storage.Tables
 require: $(this-folder)/../../readme.md
-# https://github.com/shurd/azure-rest-api-specs/tree/tablesSwagger/specification/cosmos-db/data-plane/Microsoft.TablesStorage/preview/2018-10-10
+#branch:tablesSwagger
```

Comment:
Since the commit-based URLs do not have branch information, we needed another mechanism to have the branch name, or else I have no idea how to get the latest commit hash from that branch.